internal/cmd/stack/local_preview.go

```diff
@@ -22,13 +22,25 @@ func localPreview() cli.ActionFunc {
 		}
 		ctx := context.Background()
 
+		var packagePath *string = nil
+		if cliCtx.Bool(flagProjectRootOnly.Name) {
+			root, err := getGitRepositorySubdir()
+			if err != nil {
+				return fmt.Errorf("couldn't get the packagePath: %w", err)
+			}
+			packagePath = &root
+		}
 		if !cliCtx.Bool(flagNoFindRepositoryRoot.Name) {
 			if err := internal.MoveToRepositoryRoot(); err != nil {
 				return fmt.Errorf("couldn't move to repository root: %w", err)
 			}
 		}
 
-		fmt.Println("Packing local workspace...")
+		if packagePath == nil {
+			fmt.Println("Packing local workspace...")
+		} else {
+			fmt.Printf("Packing project_root as local workspace... project_root='%s'\n", *packagePath)
```

Comment:
Maybe just replace `Packing project_root as` with `Packing '%s' as` here, without the `project_root='%s'`? It looks a bit like debug logs otherwise.